doc/classes/DisplayServer.xml

```diff
@@ -1495,60 +1496,78 @@
 		<constant name="CURSOR_MAX" value="17" enum="CursorShape">
 		</constant>
 		<constant name="WINDOW_MODE_WINDOWED" value="0" enum="WindowMode">
+			Windowed mode, i.e., [Window] doesn't occupy the whole screen (unless set to the size of the screen).
```

Comment:
We tend to follow the convention to use `e.g.` and `i.e.` without a comma after it:
```
$ rg "i\.e\. " -g'!thirdparty' -g'!*.po*' | wc -l
253
$ rg "i\.e\., " -g'!thirdparty' -g'!*.po*' | wc -l
17
$ rg "e\.g\. " -g'!thirdparty' -g'!*.po*' | wc -l
307
$ rg "e\.g\., " -g'!thirdparty' -g'!*.po*' | wc -l
6
```

So I would keep this as the convention here.

According to https://jakubmarian.com/comma-after-i-e-and-e-g/ `i.e.` without comma is British English while `i.e.,` with comma is the recommended American English style, but they point out that many Americans don't follow this style recommendation either.

Reply:
There's still a few "e.g.," added in the current commit.